internal/commands/scan.go

```diff
@@ -384,6 +387,37 @@ func scanShowSubCommand(scansWrapper wrappers.ScansWrapper) *cobra.Command {
 	return showScanCmd
 }
 
+func scanVorpalSubCommand() *cobra.Command {
+	scanVorpalCmd := &cobra.Command{
+		Use:    "vorpal",
+		Hidden: true,
+		Short:  "Run a vorpal scan",
+		Long:   "vorpal scan is able to scan a single file fast and efficiently.",
+		Example: heredoc.Doc(
+			`
+			$ cx scan vorpal --file-source <path to code file> --vorpal-latest-version <use latest Vorpal available version>
+		`,
+		),
+		Annotations: map[string]string{
+			"command:doc": heredoc.Doc(
+				`
+				https://checkmarx.com/resource/documents/en/link-to-cxone-documentation
+			`,
+			),
+		},
+		RunE: runScanVorpalCommand(),
+	}
+
+	scanVorpalCmd.PersistentFlags().Bool(commonParams.VorpalLatestVersion, false, "Set to true to update to latest version")
```

Comment:
Update Vorpal to the latest version maybe? 
Im not sure about the English here and not sure that this flag get a value

Reply:
1. We discussed it in the product group and Fatima chose this. Do you want to continue the discussion there?
2. you mean it doesn't get value in the code or by the design?
https://checkmarx.atlassian.net/wiki/spaces/AID/pages/7349010436/SAST+Lightweight+Engine+-+Plugin+Cli+Detailed+Design#1.3-Business-Case---Product-Requirements*:~:text=Flag%20%2Ds%20/,enginename%7D%2Dupdate%2Dversion

Reply:
Change phrasing for:
 "Set to true to update to latest version"****